vehicles/src/modules/special-auth/loa.service.ts

```diff
@@ -285,7 +285,6 @@ export class LoaService {
         LoaDetail,
         {
           extraArgs: () => ({
-            companyId,
```

Comment:
Is this intentional? Any corresponding changes in profiles?

Reply:
Yes, this is intentional, no change in profile needed